codegen/java-gen/src/test/scala/com/lightbend/akkasls/codegen/java/ActionTestKitGeneratorSuite.scala

```diff
@@ -34,26 +34,25 @@ import scala.util.Try
 class ActionTestKitGeneratorSuite extends munit.FunSuite {
   private val testData = TestData.javaStyle
 
+  val log = LoggerFactory.getLogger(getClass)
+
   test(
-    "it can generate an specific TestKit for the proto files " +
-    "in test/resources/testkit") {
+    "it can generate an specific TestKit for the user-function file " +
+    "in test/resources/descriptor-sets build from the proto files" +
+    "from samples/java-eventsourced-counter") {
 
     val packageName = "com.example.actions"
     val model = generateModel
-    val className = "CounterJournalToTopicAction"
     val service: ModelBuilder.ActionService = {
       model.get.services("com.example.actions.CounterJournalToTopic").asInstanceOf[ModelBuilder.ActionService]
-
-    }
-    val entity: ModelBuilder.EventSourcedEntity = {
-      model.get.entities("com.example.domain.Counter").asInstanceOf[ModelBuilder.EventSourcedEntity]
     }
+    val className = service.className
 
-    //This prints the contents of the model, services or entity. Necessary to
+    //This prints the contents of the model, services or entity. It's necessary to
     // see how the user-function.desc (see generateModel below) maps to ModelBuilder objects.
-    // println(munit.Assertions.munitPrint(entity))
+    log.debug(munit.Assertions.munitPrint(service))
```

Comment:
I think using println temporarily to inspect when changing something is fine, instead of adding test logging config etc. Once the test passes it's not super interesting with that model printed?

Reply:
Ok. I wasn't even sure myself. Adding the log file and the logger might be a bit too much 👍